jwst/ami/lg_model.py

```diff
@@ -66,17 +68,11 @@ def __init__(
         pixweight: 2D float array, default is None
             weighting array
 
-        datapath: string
-            directory for output (will remove for final)
-
         phi: float 1D array
             distance of fringe from hole center in units of waves
 
-        refdir: string
-            directory containing ref files (will remove for final)
-
-        chooseholes: list ?
-            holes ...?
+        chooseholes: list of strings
+            None, or e.g. ['B2', 'B4', 'B5', 'B6'] for a four-hole mask
```

Comment:
Can you add a note about what happens if this is None?
```suggestion
        chooseholes: list of strings or None
            E.g. ['B2', 'B4', 'B5', 'B6'] for a four-hole mask
```